scripts/assets/test_file_exemptions.textproto

```diff
@@ -49,11 +49,15 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/Develope
 exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsStarterImpl.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsStarterModule.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsViewModel.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/ForceCrashButtonClickListener.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/RouteToMarkChaptersCompletedListener.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/RouteToMarkStoriesCompletedListener.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/RouteToMarkTopicsCompletedListener.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/devoptions/RouteToViewEventLogsListener.kt"
```

Comment:
Are these exemptions missed from earlier PRs?

Reply:
No, these were actually not in the correct lexicographical order. So, made this correction in my PR.